CHANGELOG.md

```diff
@@ -3,6 +3,16 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/) and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## 8.0.0 
```

Comment:
Een changelog entry is een verwijzing naar de issue waar de scope is aangegeven.

Verder weten we nu nog niet welke versie de volgende gaat worden, in dat geval vermelden we de versie `next`. Als ik de release uitbreng zal ik `next` vervangen door een semver release.

Ik zal CONTRIBUTING.md uitbreiden met verbeterde instructies. Laat het voor nu maar, ik zal het aanpassen zodra ik later op de dag de release uitbreng (ik vermoed dat het `7.3.0` wordt)